packages/client/bin/cli.ts

```diff
@@ -402,6 +414,11 @@ const args: ClientOpts = yargs(hideBin(process.argv))
       'Skip executing blocks in new payload calls in engine, alias for --engineNewpayloadMaxExecute=0 and overrides any engineNewpayloadMaxExecute if also provided',
     boolean: true,
   })
+  .option('useJsCrypto', {
+    describe: 'Use pure Javascript cryptography functions',
+    boolean: true,
+    default: false,
+  })
```

Comment:
Hey there, looks great overall! 🤩

One question here: was this intentional to add such an option for the JS/WASM crypto here or did you just misread my suggestion to add a flag *to the EVM/VM state/blockchain test runner* (maybe re-read the original message)? 😂

Intentional or not, also not against such an option in the client as well and since we have it now we might also keep. I do think it will likely still make sense (or: we should do) to add an option to the test runner (so: here https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/vm/test/tester/index.ts#L21 ), so that we can run the VM/EVM tests with or without the WASM crypto.

Reply:
🤦 yes, completely misread your original message. Will update the test runner in my next set of changes.

Reply:
Haha, option is super handy to do performance comparisions though. 😆

I hope it's ok if I play around with this branch and do some updates and testing stuff? Or do you have something local or are working on it right now?